orbit/radius.py

```diff
@@ -616,8 +620,18 @@ def handle_dashboard(rocket):
         rev1 = asn_gradeables.where(grd_tbl.component == 'review1').first()
         rev2 = asn_gradeables.where(grd_tbl.component == 'review2').first()
         final = asn_gradeables.where(grd_tbl.component == 'final').first()
-        ret += str(AsmtTable(assignment, oopsieness, peer1, peer2, init, rev1,
-                             rev2, final))
+
+        repo = git.Repo('/var/lib/git/grading.git')
+        grades = {}
+        for component in ['review1', 'review2', 'final']:
+            tag = f'{assignment.name}_{component}_{rocket.session.username}'
+            try:
+                grades[component] = repo.git.execute(['git', 'notes', '--ref=grade', 'show', tag])
+            except git.GitCommandError:
+                grades[component] = '-'
```

Comment:
to implement my suggestion we would also change this to `grades[component] = None`

Reply:
done